### DISCUSSION TITLE
runing for wordpress website

### DISCUSSION THREAD

**Discussion by Zhekava** [CoderLine/alphaTab#1620] [Q&A]
hello
I wanted to implement in a WordPress website, so that by giving the Internet address of the guitar pro file on the page of each product, that note will be displayed. I added a field with advanced custom fields where I give the address or the name of the file. I am familiar with coding, but I don't know how to read and create a function.

**Comment by Danielku15** [comment]
Hi. 
I'd recommend you reach out to the wordpress developer community to learn how to make wordpress plugins and how to integrate JavaScript libraries in general. There should be plenty of tutorials and guides around that. At some point in the frontend some initialization function should be called where you then use alphaTab to initialize in a given DOM element.

**Reply by fanjielin** [reply]
这个问题，我可以帮你

**Comment by Ejire1** [comment]
How can i add any form under checkouts buttons if i don't want to add payment method
